nestedtensor/csrc/functions.cpp

```diff
@@ -60,5 +63,195 @@ NestedTensor squeeze(
   return NestedTensor(std::move(structure));
 }
 
+NestedTensor relu(NestedTensor input, 
+                  c10::optional<bool> inplace) {
+  TensorNode input_structure = input.get_structure();
+  TensorNode res = map([&](at::Tensor t){
+      return torch::relu(t.unsqueeze(0)).squeeze(0);
+  }, input_structure);
+
+  return NestedTensor(std::move(res));
+}
+
+NestedTensor relu_out(NestedTensor& input) {
+  input = relu(input, true);
+  return input;
+}
+
+NestedTensor dropout(NestedTensor input, 
+                     c10::optional<double> p, 
+                     c10::optional<bool> training, 
+                     c10::optional<bool> inplace) {
+  TensorNode input_structure = input.get_structure();
+  TensorNode res = map([&](at::Tensor t){
+      return torch::dropout(t, p.value(), training.value());
+  }, input_structure);
+
+  return NestedTensor(std::move(res));
+}
+
+NestedTensor conv2d(const NestedTensor input, 
+                    const at::Tensor weight, 
+                    c10::optional<at::Tensor> bias, 
+                    c10::optional<std::vector<int64_t>> stride,
+                    c10::optional<std::vector<int64_t>> padding,
+                    c10::optional<std::vector<int64_t>> dilation,
+                    c10::optional<int64_t> groups) {
+  TensorNode structure = input.get_structure();
+  auto options = F::Conv2dFuncOptions().stride(stride.value())
+                                       .padding(padding.value())
+                                       .dilation(dilation.value())
+                                       .groups(groups.value());
+  if (bias.has_value()) {
+      options = options.bias(bias.value());
+  }
+
+  TensorNode res = map([&, options](at::Tensor t){
+      return F::conv2d(t.unsqueeze(0), weight, options).squeeze(0);
+  }, structure);
+
+  return NestedTensor(std::move(res));
+}
+
+NestedTensor maxPool2d(NestedTensor input,
```

Comment:
The name used in aten is [max_pool2d](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L1814). Think of these as overloads of native functions.